### DISCUSSION TITLE
Does SingleFile is able to save as multiple file ?

### DISCUSSION THREAD

**Discussion by itwars** [gildas-lormeau/SingleFile#1117] [Q&A]
Does SingleFile is able to save as multiple file ? I need to have images (jpg,png,avif,webp,svg) in separated files, for my tests purpose
Thx

**Comment by amalgame21** [comment]
Try singleFileZ, it will save as zip file (just with .html extension), you can extract it and get the images.

**Comment by itwars** [comment]
Excellent! Thx very much for your answer.
